src/Lower.cpp

```diff
@@ -58,7 +58,7 @@ using std::map;
 using std::pair;
 using std::make_pair;
 
-Stmt lower(const vector<Function> &outputs, const Target &t, const vector<IRMutator *> &custom_passes) {
+Stmt lower(const vector<Function> &outputs, const string &pipeline_name, const Target &t, const vector<IRMutator *> &custom_passes) {
```

Comment:
Now that the pipeline name is here, I'd love to add it to the asserts that get generated. A consistent complaint I have is when I get a runtime assert failure without being able to tell which pipeline it came from. Maybe for another PR though.